client/src/components/Buttons/ButtonText.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export enum ButtonTypes {
+  Button = "button",
+  Submit = "submit",
+  React = "reset",
+}
+
+type CustomButtonProps = {
+  title: string;
+  type: ButtonTypes | undefined;
+};
+
+const CustomButton: React.FC<CustomButtonProps> = ({
+  title,
+  ...otherProps
+}) => {
+  return (
+    <button
+      className="col-span-12 py-2 border border-transparent shadow-sm text-sm font-medium rounded-md text-white bg-orange w-full"
```

Comment:
```suggestion
      className=`py-2 border border-transparent shadow-sm text-sm font-medium rounded-md text-white bg-orange ${fluid ? 'w-full' : 'w-auto'}`
```
Chỉ để style chung thôi nha, col-span-12 hay gì thì là cái thẻ div bọc ngoài quyết định
Chứ để nguyên class col-span-12 như này. Là xài button nào nó cũng sẽ chỉ có 1 size là col-span-12.
Bổ sung thêm 1 props là fluid để xác định xem cái button đó có full width hay k

<Button /> (size bình thường, width auto)
<Button fluid={true} /> (size full, width 100%)